source/connection-monitoring-and-pooling/tests/pool-clear-destroy-available-conns.yml

```diff
@@ -0,0 +1,44 @@
+version: 1
+style: unit
+description: Pool MUST be able to destroy all available connections
```

Comment:
The current wording of the spec doesn't require that the background thread is immediately scheduled when `closeInUseConnections` is false, so I think we can remove this test. Alternatively we can update the wording to always require it on pool clear, but I don't think that's necessary.

Reply:
changed wording, as it was discussed offline